plugins/mcreator-core/procedures/world_send_chat_advanced.json

```diff
@@ -11,14 +11,14 @@
       "options": [
         ["white", "white"],
```

Comment:
you can already map to rgb values here in FTL to avoid the chain of if statements